ktlint-ruleset-standard/src/main/kotlin/com/pinterest/ktlint/ruleset/standard/ImportOrderingRule.kt

```diff
@@ -216,7 +217,9 @@ public class ImportOrderingRule :
                     if (autoCorrect && canAutoCorrect) {
                         node.removeRange(node.firstChildNode, node.lastChildNode.treeNext)
                         sortedImportsWithSpaces.reduce { current, next ->
-                            node.addChild(current, null)
+                            if (!(current.isWhiteSpace() && node.firstChildNode == null)) {
```

Comment:
what is this if-check for? just tried it locally and it worked without

Reply:
If the file does not contain imports for "empty.group1" then the current node will be a WhiteSpace node containing "\n\n" which would lead to a consecutive blank line violation. Now the node is only added when a previous import was already added or when the first node actually isn't empty.

Reply:
could you add a test case for this as well? as I mentioned I tried to remove this check locally and run the tests - they passed.

Reply:
Strangely enough I can not reproduce the original problem anymore. The original code gives same output as the modified code. I propose to close the PR without merging it.

Reply:
ok, let us know if we need to reopen or you find a way to reproduce it